chat-plugins/othermetas.js

```diff
@@ -0,0 +1,98 @@
+//Other metas plugin by Spandan
+'use strict';
+
+exports.commands = {
+	mixandmega: 'mnm',
+	mnm: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		if (!target || toId(target) === "" || !target.includes('@')) return this.parse('/help mixandmega');
```

Comment:
Wait, nvm.

    if (!toId(target) || !target.includes('@')) return this.parse('/help mixandmega');